rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2528,23 +2526,32 @@ void voteFederationChange_rollback_returnsSuccessResponseCode() {
             federationSupport.voteFederationChange(firstAuthorizedTx, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
             federationSupport.voteFederationChange(secondAuthorizedTx, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
 
-            ABICallSpec rollbackAbiCallSpec = new ABICallSpec(FederationChangeFunction.ROLLBACK.getKey(), new byte[][]{
-                expectedFederatorBtcECKey.getPubKey(),
-                expectedFederatorRskKey.getPubKey(),
-                expectedFederatorMstKey.getPubKey()
-            });
+            ABICallSpec addFederationMemberAbiCallSpec = new ABICallSpec(FederationChangeFunction.ADD.getKey(), new byte[][]{expectedFederatorBtcECKey.getPubKey()});
```

Comment:
See the comments in other PRs, this is disabled at the moment

Reply:
Updated.